jsonnet/components/prometheus/prometheus-rules.jsonnet

```diff
@@ -0,0 +1,17 @@
+/*Cannot use https://github.com/prometheus/prometheus/blob/main/documentation/prometheus-mixin/mixin.libsonnet since it generates yaml with double quotes wrapped*/
+
+local rules = (
+  import 'github.com/prometheus/prometheus/documentation/prometheus-mixin/mixin.libsonnet'
```

Comment:
doesn't this contradict the comment above?

Reply:
no but I can modify the sentence. what I meant was I cannot directly use import, I do extract `.prometheusAlerts` from the import

Reply:
aha! it would be nice if you could clarify what you meant in the comment itself 👼